Localize/lang/strings.json

```
@@ -7,6 +7,7 @@
   "+JCU+Y": "Select this if you are configuring an on-prem connection",
   "+W2vTp": "Please enter an e-mail address",
```

- "please" isn't used appropriately here. See ["please" - Microsoft Style Guide](https://learn.microsoft.com/style-guide/a-z-word-list-term-collections/p/please).
- "email" doesn't use a hyphen. See ["email" - Microsoft Style Guide](https://learn.microsoft.com/style-guide/a-z-word-list-term-collections/e/email):

```suggestion
  "+W2vTp": "Enter an email address.",
```